gpustat/__main__.py

```diff
@@ -107,6 +109,13 @@ def main(*argv):
         help='Allow to print additional informations for debugging.'
     )
     args = parser.parse_args(argv[1:])
+    if args.show_all:
+        args.show_cmd = True
+        args.show_user = True
+        args.show_pid = True
+        args.show_fan_speed = True
+        args.show_power = ['draw,limit']
+        del args.show_all
```

Comment:
Is there any reason to delete the argument after using it.

Reply:
Because the API of `print_gpu` from `core.py` only takes certain arguments in the parameter list, but in `__main__.py`, `**vars(args)` is passed into the function. I just followed the implementation in line 130, where the author deletes the used `interval` argument from `args`. (The `show_all` option does not need to be passed into the function after those related arguments are set to True.)
https://github.com/wookayin/gpustat/blob/922116fd593f1579d2711b2c88f2bfa3cdef28ae/gpustat/__main__.py#L129-L131

Reply:
Thanks, LGTM.